rasa_core/policies/policy.py

```diff
@@ -28,13 +30,44 @@ def _create_featurizer(cls, featurizer=None):
         else:
             return cls._standard_featurizer()
 
+    @staticmethod
+    def _load_tf_config(config: Dict[Text, Any]) -> None:
+        """Prepare tf.ConfigProto for training"""
+        if config.get("tf_config") is not None:
+            return tf.ConfigProto(**config["tf_config"])
+        else:
+            return None
+
     def __init__(self, featurizer: Optional[TrackerFeaturizer] = None) -> None:
         self.__featurizer = self._create_featurizer(featurizer)
 
     @property
     def featurizer(self):
         return self.__featurizer
 
+    @property
+    def tf_defaults(self):
```

Comment:
we don't need self.tf_defaults anymore, because if there is no tf_config, we init tf.Session with None

Could you please create a test with this `config` and check that `tf.ConfigProto` was created correctly?

Reply:
> we don't need self.tf_defaults anymore, because if there is no tf_config, we init tf.Session with None
> 
> Could you please create a test with this `config` and check that `tf.ConfigProto` was created correctly?

Sorry, I'm afraid to create a test and I have no idea.

Reply:
no worries, I can do that, but I'll need clone your repo branch into our branch and create separate PR for it, but your commits marked as authored by you will still be there. Is it ok?

Reply:
ok, no problem. Then I can create another pr about position embedding.